packages/stimulus/src/index.ts

```diff
@@ -3,9 +3,12 @@ import type { JSClient, JSSpan } from "@appsignal/types"
 export function installErrorHandler(appsignal: JSClient, application: any) {
   const prevHandler = application.handleError
 
-  application.handleError = function (error: Error, message: string) {
```

Comment:
This space was removed by eslint auto-formatting, so I _assume_ it is consistent with the project style guide. Let me know if I need to do something different here.